src/CMakeLists.txt

```diff
@@ -836,6 +848,18 @@ endif(WITH_TESTS)
 
 add_subdirectory(compressor)
 add_subdirectory(tools)
+add_subdirectory(dmclock) # after gmock
+
+add_dependencies(tests
+  dmclock-tests dmclock-data-struct-tests)
+
+if(WITH_TESTS)
+install(PROGRAMS
+  ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/dmclock-tests
```

Comment:
might want to include this in "ceph-test". see debian/ceph-test.install and ceph.spec.in (search for `%files -n ceph-test`)